test/lib/ufe/testUIIcons.py

```diff
@@ -146,6 +144,14 @@ def testUIIcons(self):
                 ('PhysicsSphericalJoint',   'out_USD_UsdTyped.png')
             ])
 
+        # Special case for node types which are in an AL schema.
+        # They aren't available when compiling without the AL plugin.
+        if Usd.SchemaRegistry.IsConcrete(Tf.Type.FindByName('AL_usd_FrameRange')):
+            primTypes.extend([
+                ('ALExamplePolyCubeNode',   'out_USD_UsdTyped.png'),
+                ('ALFrameRange',            'out_USD_UsdTyped.png')
+        ])
```

Comment:
If you build locally without the AL plugin you won't have the schema that defines these two node types.